### ISSUE TITLE
make-json created 0 files

### ISSUE THREAD

**Issue by woro83c** [wp-cli/i18n-command#136]
heya, i'm running the `make-json` command without any luck. within my `/languages` folder i have the following files:

```zsh
my-plugin-en_AU.mo
my-plugin-en_AU.po
my-plugin.mo
my-plugin.pot
```

then when i run `wp i18n make-json languages`, i get the following:

```zsh
Success: Created 0 files.
```

**Comment by swissspidy** [comment]
This will happen if your PO file doesn‘t contain any strings coming from JavaScript files. Please verify if that is the case.

**Comment by woro83c** [comment]
that removes the success message, but it still leaves me with:

```zsh
Fatal error: Uncaught InvalidArgumentException: The language "en_AU" is not valid in phar:///usr/local/bin/wp/vendor/gettext/gettext/src/Translations.php:356
```

which has also been raised here:
https://github.com/swissspidy/gutenberg-i18n-block/issues/6

thanks for your help!

**Comment by swissspidy** [comment]
That‘s gonna be fixed by #133.

**Comment by woro83c** [comment]
ah great, thank you!
